### PR TITLE
now we can connect to both idendities

### PR DESCRIPTION
**Pull request by alexradzin**

### CODE REVIEW

File: deps.edn [hunk 1]
```diff
@@ -2,9 +2,9 @@
  ["src" "resources"]
 
  :deps
- {com.firebolt/firebolt-jdbc {:mvn/version "3.0.0-SNAPSHOT"}} ;TODO: Change to the JDBC version supported new identity
+ {io.firebolt/firebolt-jdbc {:mvn/version "3.0.1"}} ;TODO: Change to the JDBC version supported new identity
```
Comment (ptiurin) [inline deps.edn]: I think we can remove the TODO here?

File: resources/metabase-plugin.yaml [hunk 2]
```diff
@@ -11,19 +11,19 @@ driver:
...
   connection-properties:
-    - name: client_id
-      display-name: Client ID
+    - name: user
+      display-name: User/Client ID
       required: true
-    - name: client_secret
-      display-name: Client Secret
+    - name: password
+      display-name: Password/Client Secret
```
Comment (ptiurin) [inline resources/metabase-plugin.yaml]: I believe the product decision on parameter naming was to keep the `Client ID` / `Client Secret` even in driver that support both, but add a note in V1 docs to use username/password in those fields.

Comment (alexradzin) [reply resources/metabase-plugin.yaml]: You are absolutely right but current logic required from my in jdbc driver does not allow this. I spent an hour trying to understand how to make metabase driver to support both v1 and v2 and this was the solution. Please take a look on implementation of method [getUrlVersion()](https://github.com/firebolt-db/jdbc/blob/0a4331a0d9ade4f62050266ad671f47a1f39d584/src/main/java/com/firebolt/jdbc/connection/FireboltConnection.java#L124).

Using user/password + "old" URL format makes it possible to support both v1 and v2.

File: deps.edn (hunk 1 above)
Comment (alexradzin) [reply deps.edn]: done

File: resources/metabase-plugin.yaml (hunk 2 above)
Comment (ptiurin) [reply resources/metabase-plugin.yaml]: Sorry for the confusion, I don't actually mean the parameter name or format. What I mean is, `display-name:` shouldn't have `User/` or  `Password/` prepended in the UI. Using username and password would only be described in the V1 docs, from a new user standpoint who just installs our driver there's only client id and client secret.

Comment (alexradzin) [reply resources/metabase-plugin.yaml]: I see. I can remove this but I am afraid it can confuse the existing customer(s) that (still?) use v1

Comment (ptiurin) [reply resources/metabase-plugin.yaml]: It's not something I've decided - the product team deemed it an acceptable risk so the decision was to change it.

Comment (alexradzin) [reply resources/metabase-plugin.yaml]: NP, renamed back to client_id and client_secret

**Review by ptiurin — APPROVED** [review]

**Pull request merged**